extensions/wikia/ArticleVideo/bower_components/html5-skin/tests/components/utils-test.js

```diff
@@ -1,5 +1,13 @@
 jest.dontMock('../../js/components/utils');
+jest.dontMock('deepmerge');
+jest.dontMock('../../config/skin');
+
 var Utils = require('../../js/components/utils');
+var DeepMerge = require('deepmerge');
+var SkinJSON = require('../../config/skin');
+OO = {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add the "let", "const" or "var" keyword to this declaration of "OO" to make it explicit. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2703)